src/main/java/omero/gateway/Gateway.java

```diff
@@ -1154,7 +1154,53 @@ private boolean isSessionID(String s) {
         }
         return true;
     }
-    
+
+    /**
+     * Test if server and client versions are compatible.
+     * Not {@code private} only to allow unit testing.
```

Comment:
Does that mean users can or can't use it?

Reply:
Shouldn't consider it part of supported API anyway. It's also not `public`; this is really a note to developers who might wonder why it's not `private`.